pkg/platform/platform.go

```diff
@@ -15,11 +15,10 @@ package platform
 
 // Platform abstracts all operations which are handled by the specific platform (e.g. Kubernetes).
 type Platform interface {
-	CreateService(name, targetApp string, port, targetPort uint16)
-	UpdateService(name, targetApp string, port, targetPort uint16)
-	DeleteService(name string)
-	CreateEndpoint(name, targetIP string, targetPort uint16)
-	UpdateEndpoint(name, targetIP string, targetPort uint16)
-	DeleteEndpoint(name string)
+	CreateService(name, host, targetApp string, port, targetPort uint16)
+	UpdateService(name, host, targetApp string, port, targetPort uint16)
+	DeleteService(name, host string)
+	CreateExternalService(name, host, ExternalName string)
+	UpdateExternalService(name, host, ExternalName string)
```

Comment:
confirming: there's no corresponding `DeleteExternalService` since it uses the `DeleteService` call, righ?

Reply:
Yes